### DISCUSSION TITLE
关于PCL经常不见那些事

### DISCUSSION THREAD

**Discussion by MangoGong** [Meloong-Git/PCL#1804] [提问] [answered]
每次PCL内测版更新的时候都会自己不见，**本人已经赞助22.33元的档位，且确信火绒、Windows defender没有删除PCL**。
想问一下为什么会出现这样的情况。
谢谢
2023/05/07

**Comment by WForst-Breeze** [comment]
参考 #666，实际应该就是 Defender 导致，现在 Defender 会静默删除 PCL2 而不留任何提示记录或痕迹信息（只能使用行为监测工具捕捉到）。原因为快照版并没有向微软上传误报，正式版无此问题，您可以将 PCL2 添加至 Defender 排除项中后查看问题是否依然存在

**Reply by iPodtouch6557** [reply]
补充：而且关闭了defender有可能他还会重启，想要问题解决真的全靠运气……

**Reply by WForst-Breeze** [reply, marked as answer]
直接添加排除项应该就可以解决大部分此类问题，我这里一直就没再出现此类问题，正常情况下关闭 Defender 稍稍有些多此一举。不过有些设备的环境较为特殊，具体打算如何操作还得看用户个人的选择（

**Reply by MangoGong** [reply]
> 参考 #666，实际应该就是 Defender 导致，现在 Defender 会静默删除 PCL2 而不留任何提示记录或痕迹信息（只能使用行为监测工具捕捉到）。原因为快照版并没有向微软上传误报，正式版无此问题，您可以将 PCL2 添加至 Defender 排除项中后查看问题是否依然存在

可是装了火绒照理说Windows defender是不会再介入了，可不可能是别的原因？

**Reply by WForst-Breeze** [reply]
从个人实际经历来看，安装了火绒后我曾也出现了 PCL2 意外消失的问题，但当我将其添加进排除项后问题就的确再也没出现了。#666 中的大部分用户也都是安装了火绒但问题依然存在的，您先把 PCL2 加入排除项后使用一段时间看看？有什么进一步问题可以再补充

**Reply by MangoGong** [reply]
> 从个人实际经历来看，安装了火绒后我曾也出现了 PCL2 意外消失的问题，但当我将其添加进排除项后问题就的确再也没出现了。#666 中的大部分用户也都是安装了火绒但问题依然存在的，您先把 PCL2 加入排除项后使用一段时间看看？有什么进一步问题可以再补充

好的，感谢！

**Comment by hejiehao** [comment]
我也有过，把正式版和快照版放在同一个文件夹，正式版的文件名为Plain Craft Launcher 2.exe，快照版的文件名为Plain Craft Launcher 2 snapshot.exe，快照版一更新Plain Craft Launcher 2 snapshot.exe就没了，启动器重启之后也没有启动起来
